image/Dockerfile-stack

```diff
@@ -1,35 +1,34 @@
 FROM node:12
 
 ENV APPSODY_PROJECT_DIR=/project
-ENV APPSODY_MOUNTS=/:/project/user-app
-ENV APPSODY_DEPS=/project/user-app/node_modules
+ENV APPSODY_MOUNTS=.:/project/function
+ENV APPSODY_DEPS=/project/function/node_modules
 
-ENV APPSODY_WATCH_DIR=/project/user-app
-ENV APPSODY_WATCH_IGNORE_DIR=/project/user-app/node_modules
+ENV APPSODY_WATCH_DIR=/project/function
+ENV APPSODY_WATCH_IGNORE_DIR=/project/function/node_modules
 ENV APPSODY_WATCH_REGEX="^.*.js$"
 
+ENV APPSODY_TEST="npm test && npm test --prefix function"
+ENV APPSODY_TEST_ON_CHANGE=""
+ENV APPSODY_TEST_KILL=false
+
 ENV APPSODY_RUN="npm start"
-ENV APPSODY_RUN_ON_CHANGE="npm start"
-ENV APPSODY_RUN_KILL=true
```

Comment:
I've read the [appsody docs](https://appsody.dev/docs/stacks/environment-variables#run-environment-variables), and it's not clear to me that removing these results in the same behavior. I think it's probably best to keep these explicit.

Reply:
This would indeed change behavior if omitted, but that onchange command is still there (line 16), just with a parameterized value to be explicit it is essentially an alias of APPSODY_RUN.
The value of the "kill" command, however, is actually mentioned in the docs as being default true.  I added it back anyway to be explicit as line 17.